omero_figure/templates/figure/index.html

```diff
@@ -581,14 +581,19 @@ <h4>Dynamic properties</h4>
                             <td><i>Height</i>: <strong>300</strong></td>
                         </tr>
                         <tr>
-                            <th>Time with decimal precision of 2</th>
-                            <td>Time (s): [time.seconds.2]</td>
-                            <td>Time (s): 0.93 s</td>
+                            <th>Width with decimal precision of 3</th>
+                            <td>Width: [width.unit.3]</td>
+                            <td>Width: 4.450 µm</td>
                         </tr>
                         <tr>
                             <th>Time with frame n°3 as the "zero"</th>
-                            <td>Time (ms): [time.milliseconds-3]</td>
-                            <td>Time (ms): -58 ms</td>
+                            <td>Time: [time.milliseconds-3]</td>
+                            <td>Time: -58 ms</td>
+                        </tr>
+                        <tr>
+                            <th>Time with decimal precision (2) and "zero" (3)</th>
+                            <td>Time: [time.seconds.2-3]</td>
+                            <td>Time: -0.058 s</td>
```

Comment:
"decimal precision (2)" should be `Time: -0.06 s`?

Reply:
oh yes indeed